examples/tracing-compare.cpp

```diff
@@ -73,7 +73,8 @@ std::unique_ptr<CompiledFunction> compileModel(Module &module,
 
   llvm::outs() << "Starting compile.\n";
   backend->optimizeFunction(CompilationMode::Infer, F);
-  return backend->instrumentAndCompile(F);
+  CompileOptions opts{true, true};
```

Comment:
I'm slightly annoyed by this form of the initialization. If you look at it, you don't really know which options were initialized. Could you use a more explicit syntax with field names?

Reply:
Sure thing, just trying to save on lines I guess but this isn't super readable (plus it doesn't work on g++ 4.8 as it turns out).